sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -253,9 +256,18 @@ def tune(
                 to the base image packages. These packages are installed before
                 executing the objective function.
             pip_index_url: The PyPI url from which to install Python packages.
-            metrics_collector_config: Specify the config of metrics collector,
-                for example, `metrics_collector_config = {"kind": "Push"}`.
-                Currently, we only support `StdOut` and `Push` metrics collector.
+            
+            `metrics_collector_config`: Specify the configuration for the metrics collector with following keys:
+            - **kind**: Specify the kind of Metrics Collector. Currently supported values are:
+                - `StdOut`: Collects metrics from standard output.
+                - `None`: No metrics collection.
```

Comment:
This is not supported.

Reply:
Thanks for pointing this out. Can you lead me to where I can find all supported metric collector. For the current comment I had referenced `/pkg/ui/v1beta1/frontend/src/app/models/experiment.k8s.model.ts`

Reply:
Yeah, UI also needs to be updated with the latest changes cc @Electronic-Waste 
Please ref the official CRDs APIs for Metrics Collector spec: https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/common/v1beta1/common_types.go#L207-L227

Reply:
Thanks Andrey, I'll update UI with the latest changes.